addons/AdherentBuffs/AdherentBuffs.lua

```diff
@@ -0,0 +1,112 @@
+-- Copyright © 2015, Mafai, Sechs
+-- All rights reserved.
+-- 
+-- Redistribution and use in source and binary forms, with or without
+-- modification, are permitted provided that the following conditions are met:
+-- 
+--     * Redistributions of source code must retain the above copyright
+--       notice, this list of conditions and the following disclaimer.
+--     * Redistributions in binary form must reproduce the above copyright
+--       notice, this list of conditions and the following disclaimer in the
+--       documentation and/or other materials provided with the distribution.
+--     * Neither the name of AdherentBuffs nor the
+--       names of its contributors may be used to endorse or promote products
+--       derived from this software without specific prior written permission.
+-- 
+-- THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+-- ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+-- WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+-- DISCLAIMED. IN NO EVENT SHALL <your name> BE LIABLE FOR ANY
+-- DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+-- (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+-- LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+-- ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+-- (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+-- SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ 
+config = require ('config')
+ 
+_addon.name     = 'AdherentBuffs'
+_addon.author   = 'Mafai, Sechs'
+_addon.version  = '1.1'
+_addon.commands = {'adherentbuffs','ab'}
+ 
+defaults = T{}
+ 
+defaults.AnnounceMode = 'party' --this can be say/party/linkshell/linkshell2/shout/s/p/l/l2/sh
+defaults.AutoAnnounce = 'off' --this can be true/t/on/false/f/off
+ 
+settings = config.load(defaults)
+ 
+adherent_maps = {['Steadfast Adherent']="PLD, DEF+", ['Furtive Adherent']="WHM, MDB+", ['Occult Adherent']="WAR, EVA+",
+		['Fleet Adherent']="WAR, Haste+", ['Brawny Adherent']="DRK, ATK+", ['Martial Adherent']="DRK,Regain+",
+		['Honed Adherent']="RDM, Fast Cast+", ['Insidious Adherent']="RDM, MEVA+", ['Hexbreaking Adherent']="BLM, MAB+"}
+chatmodes = S{'say','party','linkshell','linkshell2','shout','s','p','l','l2','sh'}
+automodes = S{'true','t','on','false','f','off'}
+moblist = S{}
+		
+windower.register_event('addon command', function (command,...)
+	command = command and command:lower()
```

Comment:
There used to be "or 'help'" in there, I removed it because I didn't want to default to Help but... that would be a strange behaviour compared to other Windower Addons, so I take your point. Will revert it back and add that or once more.